pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -170,8 +182,10 @@ def handle_NLP_subproblem_optimal(fixed_nlp, solve_data, config):
                              config)
         add_affine_cuts(solve_data, config)
     elif config.strategy == 'PSC':
+        # !!THIS SEEMS LIKE A BUG!! - mrmundt #
```

Comment:
@jsiirola  - `add_psc_cut` and `add_gbd_cut` are not defined anywhere (in all of `pyomo` - I checked), so if these statements are triggered, there will be a failure.

Reply:
@bernalde: thoughts?

Reply:
That is right, there was an ancient implementation that got removed from the code base after a major MindtPy rewrite. We will remove the GBD and PSC strategies from MindtPy until these methods are reimplemented @ZedongPeng 
For the moment you can leave the comment. It will remind us of that.